### PR TITLE
focus latex textarea on mount if it is empty

### PR DESCRIPTION
**Pull request by Entkenntnis**
This will autofocus textarea when using strg-m

### CODE REVIEW

File: packages/ui-renderer/src/components/textarea.tsx [hunk 1]
```diff
@@ -29,11 +29,15 @@ const StyledIgnoreKeys = styled(IgnoreKeys)({
...
+  props: Omit<TextareaAutosizeProps, 'as' | 'ref'> & {
+    inputRef?:
+      | React.RefObject<HTMLTextAreaElement>
+      | ((node: HTMLTextAreaElement) => void)
+  }
 ) {
   return (
     <StyledIgnoreKeys>
-      <Textarea {...props} />
+      <Textarea {...props} inputRef={props.inputRef} />
```
Comment (Knorrke) [inline packages/ui-renderer/src/components/textarea.tsx]: this doesn't change anything.

File: packages/ui-renderer/src/components/textarea.tsx [hunk 2]
```diff
@@ -29,11 +29,15 @@ const StyledIgnoreKeys = styled(IgnoreKeys)({
 })
 
 export function EditorTextarea(
-  props: Omit<TextareaAutosizeProps, 'as' | 'ref'>
+  props: Omit<TextareaAutosizeProps, 'as' | 'ref'> & {
+    inputRef?:
+      | React.RefObject<HTMLTextAreaElement>
+      | ((node: HTMLTextAreaElement) => void)
+  }
```
Comment (Knorrke) [inline packages/ui-renderer/src/components/textarea.tsx]: from the usage below I'd guess, that inputRef is already specified correctly in `TextareaAutosizeProps`

File: packages/ui-renderer/src/components/textarea.tsx (hunk 1 above)
Comment (Entkenntnis) [reply packages/ui-renderer/src/components/textarea.tsx]: yes, that's right xD

File: packages/ui-renderer/src/components/textarea.tsx (hunk 2 above)
Comment (Entkenntnis) [reply packages/ui-renderer/src/components/textarea.tsx]: 👍

**Comment by Entkenntnis** [comment]
thanks for the feedback, I've simplified the code

**Pull request merged**